cffi_src/factory.go

```diff
@@ -135,10 +136,54 @@ func BuildRequest(input RequestInput) (*http.Request, *TLSClientError) {
 }
 
 // BuildResponse constructs a client response from a given HTTP response. The client response can then be sent to the interface consumer.
-func BuildResponse(sessionId string, withSession bool, resp *http.Response, cookies []*http.Cookie, isByteResponse bool) (Response, *TLSClientError) {
+func BuildResponse(sessionId string, withSession bool, resp *http.Response, cookies []*http.Cookie, input RequestInput) (Response, *TLSClientError) {
 	defer resp.Body.Close()
+	isByteResponse := input.IsByteResponse
+	var respBodyBytes []byte
+	var err error
+
+	if input.StreamOutputPath != nil {
+		f, err := os.OpenFile(*input.StreamOutputPath, os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0600)
```

Comment:
just curious: did you test that under windows? 
as i'm only using mac i rarely test under windows :-D

Reply:
Yes I have tested it for the chatgpt program. It works just fine.